src/main/java/spring/impls/SpringApp.java

```diff
@@ -0,0 +1,32 @@
+package spring.impls;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class SpringApp {
+    public static void main(String[] args) {
+        ApplicationContext context = new ClassPathXmlApplicationContext("ApplicationContext.xml");
```

Comment:
Let's use  ConfigurableApplicationContext instead of ApplicationContext.
In this case, you don't need to cast (see line 30)